src/arrayMethodMap.js

```diff
@@ -5,7 +5,13 @@
  */
 function applyCustomMap() {
   [].__proto__.map2 = function(callback) {
-    // write code here
+    let newArray = [];
```

Comment:
newArray is not very informative. It's not clear what is stored in newArray. `res`, `result` would be more infromative.